### PR TITLE
Departmental Skirts : Cargo

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Adds jumpskirts for cargo techs and the quartermaster.

## Why It's Good For The Game

More uniform variety. Plus they already wear shorts, so why not skirts?

## Changelog
:cl:
add: Cargo skirts
imageadd: Sprites for cargo skirts
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/jobs/cargo.dm [hunk 1]
```diff
@@ -15,6 +25,16 @@
 	mutantrace_variation = MUTANTRACE_VARIATION
 	alt_covers_chest = TRUE
 
+/obj/item/clothing/under/rank/cargo/tech/skirt
+	name = "cargo technician's jumpskirt"
+	desc = "Skiiiiirts! They're comfy and easy to wear"
```
Comment (utilbird) [inline code/modules/clothing/under/jobs/cargo.dm]:
```suggestion
	desc = "Skiiiiirts! They're comfy and easy to wear!"
```

**Comment by ghost** [comment]
inb4 erotic content

**Comment by AnCopper** [comment]
Beestation players who have never seen a Woman in their life on their way to explain why skirts are erotic content.

**Comment by Sinestia** [comment]
What's more common than horny really is people whining about muh


"""realism"""

**Review by utilbird — APPROVED** [review]

**Comment by MCterra10** [comment]
remirror

**Pull request merged**